doc/ADVANCED.md

```diff
@@ -56,6 +56,23 @@ Following our [basic example](BASICS.md#a-basic-example), you will run gitarro a
 gitarro.rb -r openSUSE/gitarro -c "ruby-test" -d "ruby-gitarro-tuto" -g /tmp -t /tmp/tests.sh -f ".rb" -C
 ```
 
+# Caching Http in gitarro.
+
+Gitarro by default use faraday_http cache for storing and sparing rate limiting in github API.
+
+The cache dir is saved by default in /tmp/gitarro (but you can add a custom path via option).
+
+If you enable the option --changed_since, the cache will not generated.
+This 2 options are not compatible, since --changed_since (which is a trigger option, need to consume a githup api to have the reponse time).
```

Comment:
These two options are not compatible, since ```--changed_since``` (which is a trigger option) need to consume a GitHub API to have the response time.